source/includes/harvest/_introduction.md

```diff
@@ -172,6 +172,7 @@ The timestamps below are Eastern Time.
 
 | Date                    | Description                                                                                                                       |
 | ----------------------- | --------------------------------------------------------------------------------------------------------------------------------- |
+| May 28, 2020 05:00:00PM | Candidate photos have been removed from Greenhouse and can no longer be anonymized in [PUT Anonymize Candidates](#put-anonymize-candidate) |
```

Comment:
minor suggestion: change "can no longer be anonymized" to "will no longer be accepted as a field to be anonymized"